Automattic/jetpack
# Remove temporary Newspack block override css

Fixes #20305

#### Changes proposed in this Pull Request:
* Removes temporary CSS overrides from carousel module

#### Does this pull request change what data or activity we track or use?
No

#### Testing instructions:
* Checkout PR and run in locally dev env along with [latest release of Newspack blocks](https://github.com/Automattic/newspack-blocks/releases/tag/v1.35.0)
* Enable Jetpack carousel
* Add a post of page and add a Newspack post carousel block and a core gallery block and make sure that post carousel and the gallery carousel display as expected in editor and frontend - particularly the Newspack block previous and next buttons:
<img width="643" alt="Screen Shot 2021-08-30 at 10 11 28 AM" src="https://user-images.githubusercontent.com/3629020/131267007-4f8ae53a-a94c-466b-a3e9-3d9805880e57.png">
* Repeat the above with the WPcom diff
